Python/pywarpx/WarpX.py

```diff
@@ -40,6 +41,7 @@ def create_argv_list(self, **kw):
         argv += warpx.attrlist()
         argv += my_constants.attrlist()
         argv += amr.attrlist()
+        argv += amrex.attrlist()
```

Comment:
I needed the amrex bucket to set flags for managing the arena in the Python input files when using CUDA in WSL.  These lines were commented out in the PICMI input files when preparing for CI tests in OMP.  This could be removed and pushed through as a different PR.